src/pages/home.rs

```diff
@@ -21,24 +23,74 @@ pub fn Home(cx: Scope) -> impl IntoView {
                 </div>
             </div>
         }.into_any()),
-        (String::from("help"), view! {cx, <p><span class="text-pastel-blue">"❯ "</span>"help"</p>}.into_any()),
     ]);
 
+    // Callbacks
+    let increment_level = move || {
+        set_level(level() + 1);
```

Comment:
I would change this to `level.update(|n| *n += 1)` as it's the recommended way to update signals.